presentation/src/main/java/com/yapp/presentation/ui/member/signup/Team.kt

```diff
@@ -0,0 +1,120 @@
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.yapp.common.theme.AttendanceTypography
+import com.yapp.common.yds.*
+import com.yapp.presentation.R
+
+@Composable
+fun Team(
+    viewModel: TeamViewModel = hiltViewModel(),
+    navigateToMainScreen: () -> Unit
+) {
+    val uiState by viewModel.uiState.collectAsState()
+    val TEAM_NOT_SELECTED = -1
+
+    Scaffold(
+        topBar = { YDSAppBar(onClickBackButton = {}) },
+        modifier = Modifier
+            .fillMaxSize()
+    ) {
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(horizontal = 24.dp)
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .align(Alignment.TopStart),
+            ) {
+                Spacer(modifier = Modifier.height(32.dp))
+                Text(
+                    text = stringResource(R.string.member_signup_choose_team),
+                    style = AttendanceTypography.h1,
+                    color = Color.Black
+                )
+                Spacer(modifier = Modifier.height(28.dp))
+                TeamOption(
+                    uiState,
+                    onTeamChoosed = { viewModel.setEvent(TeamContract.TeamUiEvent.ChooseTeam(it)) })
+                Spacer(modifier = Modifier.height(52.dp))
+                if (uiState.myTeam.team.isNotEmpty()) {
+                    TeamNumberOption(
+                        uiState,
+                        onTeamNumberChoosed = {
+                            viewModel.setEvent(
+                                TeamContract.TeamUiEvent.ChooseTeamNumber(it)
+                            )
+                        })
+                }
+            }
+
+            YDSButtonLarge(
+                text = stringResource(R.string.member_signup_team_confirm),
+                modifier = Modifier
+                    .padding(bottom = 40.dp)
+                    .height(60.dp)
+                    .align(Alignment.BottomCenter),
+                state = if ((uiState.myTeam.team.isNotEmpty()) and (uiState.myTeam.count != TEAM_NOT_SELECTED)) YdsButtonState.ENABLED else YdsButtonState.DISABLED,
+                onClick = { if ((uiState.myTeam.team.isNotEmpty()) and (uiState.myTeam.count != -1)) navigateToMainScreen() },
+            )
+        }
+    }
+}
+
+@Composable
+fun TeamOption(uiState: TeamContract.TeamUiState, onTeamChoosed: (String) -> Unit) {
+    val rowNum = 2
+    Column() {
+        repeat(uiState.teams.size / rowNum) { row ->
+            Row() {
+                repeat(rowNum) { index ->
+                    val team = uiState.teams[rowNum * row + index]
+                    YDSChoiceButton(
+                        text = team.team,
+                        modifier = Modifier.padding(end = 12.dp),
+                        state = if (uiState.myTeam.team == team.team) YdsButtonState.ENABLED else YdsButtonState.DISABLED,
```

Comment:
```suggestion
                        state = if (uiState.selectedTeam.team == team.team) YdsButtonState.ENABLED else YdsButtonState.DISABLED,
```
"내가 현재 선택한 팀" 이라고 표현하는게 더 좋지 않을까 싶어 제안드려요